asset-transfer-basic/application-gateway-java/src/main/java/application/java/App.java

```diff
@@ -0,0 +1,251 @@
+                    transferAssetAsync(contract);
+                    System.out.println();
+
+                    //readAssetByID
+                    System.out.println("Read Asset By ID");
+                    readAssetById(contract);
+                    System.out.println();                    
+
+                    //updateNonExistentAsset
+                    System.out.println("Update a non-existent Asset");
+                    updateNonExistentAsset(contract);
+                    System.out.println();                    
+
+                    
+
+		} catch(Throwable e){
+		  e.printStackTrace();
+		} finally {
+                    channel.shutdownNow().awaitTermination(5, TimeUnit.SECONDS);
+
+                }
+                System.exit(0);
+
+	}
+	private static void initLedger(Contract contract) throws GatewayException, CommitException {
+        
+
+        String timestamp = LocalDateTime.now().toString();
+
+        System.out.println("Submitting \"InitLedger\" transaction with arguments: time, " + timestamp);
+
+        // Submit a transaction, blocking until the transaction has been committed on the ledger.
+        byte[] submitResult = contract.submitTransaction("InitLedger", "time", timestamp);
+
+        System.out.println("Submit result: " + new String(submitResult, StandardCharsets.UTF_8));
+        System.out.println("Evaluating \"InitLedger\" query with arguments: time");
+
+        byte[] evaluateResult = contract.evaluateTransaction("InitLedger", "time");
+        System.out.println("Query result: " + new String(evaluateResult, StandardCharsets.UTF_8));
+    }
+	
+    private static void getAllAssets(Contract contract) throws GatewayException, CommitException {
+
+        String timestamp = LocalDateTime.now().toString();
+        System.out.println("Submitting \"getAllAssets\" transaction with arguments: time, " + timestamp);
+
+        // Submit a transaction, blocking until the transaction has been committed on the ledger.
+        byte[] result = contract.evaluateTransaction("getAllAssets", "time", timestamp);
+
+        System.out.println("Query result: " + new String(result, StandardCharsets.UTF_8));
+        System.out.println("Evaluating \"getAllAssets\" query with arguments: time");
+
+    }
+
+    private static void createAsset(Contract contract) throws GatewayException, CommitException {
+
+        String timestamp = LocalDateTime.now().toString();
+        String assetId = new String("asset" + timestamp);
+
+        System.out.println("Submitting \"put\" transaction for createAssetwith arguments: time, " + timestamp);
+
+        // Submit a transaction, blocking until the transaction has been committed on the ledger.
+        
+        byte[] submitResult = contract.submitTransaction("createAsset", assetId, "yellow","5", "Tom", "1300","time", timestamp);
+
+        System.out.println("Submit result: " + new String(submitResult, StandardCharsets.UTF_8));
```

Comment:
Console output needs to match other language samples (and tutorial documentation).